follower/consensus_follower.go

```diff
@@ -0,0 +1,108 @@
+package consensus_follower
+
+import (
+	"context"
+
+	"github.com/reactivex/rxgo/v2"
+
+	"github.com/onflow/flow-go/cmd"
+	access "github.com/onflow/flow-go/cmd/access"
+	"github.com/onflow/flow-go/consensus/hotstuff/notifications/pubsub"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+func finalizedBlockProducer(anb *access.UnstakedAccessNodeBuilder) rxgo.Producer {
+	return func(ctx context.Context, next chan<- rxgo.Item) {
+		anb.FinalizationDistributor.AddOnBlockFinalizedConsumer(onBlockFinalizedConsumer(next))
+
+		select {
+		case <-anb.Ready():
+			// log: component startup complete
+		case <-ctx.Done():
+			// log: component startup aborted
+			return
+		}
+
+		<-ctx.Done()
+
+		// log: shutting down
+
+		<-anb.Done()
+	}
+}
+
+func onBlockFinalizedConsumer(next chan<- rxgo.Item) pubsub.OnBlockFinalizedConsumer {
+	return func(finalizedBlockID flow.Identifier) {
+		next <- rxgo.Of(finalizedBlockID)
+	}
+}
+
+func buildAccessNode(accessNodeOptions []access.Option) *access.UnstakedAccessNodeBuilder {
+	anb := access.FlowAccessNode(accessNodeOptions...)
+	nodeBuilder := access.NewUnstakedAccessNodeBuilder(anb)
+
+	nodeBuilder.BuildConsensusFollower()
```

Comment:
this will missout on calling `Initalize` which initializes the unstaked network.

Reply:
You're right, I forgot to add `Initialize` here :)